nucypher/cli/commands/worklock.py

```diff
@@ -219,14 +212,32 @@ def claim(general_config, worklock_options, registry_options, force, hw_wallet):
                                                                 registry=registry,
                                                                 show_balances=True)
 
-    # TODO: Show amount of tokens to claim
+    bidder = worklock_options.create_bidder(registry=registry, hw_wallet=hw_wallet)
 
+    unspent_bid = bidder.available_compensation
+    if unspent_bid:
+        emitter.echo(f"Before claiming your tokens, note that WorkLock didn't use your bid completely.\n"
+                     f"An unspent amount of {prettify_eth_amount(unspent_bid)} is available for refund.")
```

Comment:
If using the suggestion below, instead of having "Before claiming..." twice, this could be modified to:

```suggestion
        emitter.echo(f"Note that WorkLock did not use your entire bid due to a maximum claim limit.\n"
                     f"Therefore, an unspent amount of {prettify_eth_amount(unspent_bid)} is available for refund.")
```